GitExternal.cmake

```diff
@@ -112,6 +112,9 @@ function(GIT_EXTERNAL DIR REPO tag)
       RESULT_VARIABLE nok ERROR_VARIABLE error WORKING_DIRECTORY "${DIR}")
     if(nok)
       message(FATAL_ERROR "git checkout ${TAG} in ${DIR} failed: ${error}\n")
```

Comment:
in 103 it's not an error for optional externals. in 106 there's a fatal error. In doubt ask the original author :)